shell/key-bindings.bash

```diff
@@ -48,15 +48,15 @@ __fzf_cd__() {
   dir=$(set +o pipefail; eval "$cmd" | FZF_DEFAULT_OPTS="$opts" $(__fzfcmd)) && printf 'builtin cd -- %q' "$dir"
 }
 
-if command -v perl > /dev/null; then
+if command -v perl >&- ; then
```

Comment:
Probably not an issue here, but wouldn't it be safer to redirect to `/dev/null`? Because we can't always be sure how the program will behave when writing to a closed stream.

```sh
$ ls >&-
$ command -v ls >&-
$ echo hello >&-
-bash: echo: write error: Bad file descriptor
$ echo hello > /dev/null
$
```
```